src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToTimestampTransformer.php

```diff
@@ -29,20 +29,19 @@ class DateTimeToTimestampTransformer extends BaseDateTimeTransformer
      * @return int A timestamp
      *
      * @throws TransformationFailedException If the given value is not an instance
-     *                                       of \DateTime or if the output
-     *                                       timezone is not supported.
+     *                                       of \DateTime.
      */
-    public function transform($value)
+    public function transform($dateTime)
```

Comment:
This needs to be fixed in a way to match the current name in the doc block, otherwise the doc block name should be changed. I did it so it's consistent with all the other DateTime transformers.

I don't understand why do you say it makes it harder to merge if it can be applied in all current maintained branches ?